documentation-site/components/yard/config/button-group.ts

```diff
@@ -1,9 +1,10 @@
 import {ButtonGroup, MODE} from 'baseui/button-group';
 import {Button} from 'baseui/button';
 import {PropTypes} from '../const';
+import {TConfig} from '../types';
 
-export default {
-  extraImports: {
+const ButtonGroupConfig: TConfig = {
+  importsConfig: {
```

Comment:
nit: I don't think it's necessary to label this as a `config` since this whole file is a config already